source/includes/fact-automatic-objectid.rst

```diff
@@ -0,0 +1 @@
+If you do not provide an :manual:`ObjectId </reference/bson-types/#objectid>, |compass-short| automatically generates an ObjectId.
```

Comment:
[small update] I would add:

"If you do not provide an ObjectId _in your document_ ... "